op-challenger/runner/metrics.go

```diff
@@ -68,34 +74,30 @@ func NewMetrics(runConfigs []RunConfig) *Metrics {
 			Name:      "success_total",
 			Help:      "Number of VM executions that successfully verified the output root",
 		}, []string{"type"}),
-		failuresTotal: factory.NewCounterVec(prometheus.CounterOpts{
+		setupFailuresTotal: factory.NewCounterVec(prometheus.CounterOpts{
 			Namespace: Namespace,
-			Name:      "failures_total",
-			Help:      "Number of failures to execute a VM",
+			Name:      "setup_failures_total",
+			Help:      "Number of setup failures before VM execution",
 		}, []string{"type"}),
-		consecutiveFailuresCurrent: factory.NewGaugeVec(prometheus.GaugeOpts{
+		consecutiveSetupFailures: factory.NewGaugeVec(prometheus.GaugeOpts{
 			Namespace: Namespace,
-			Name:      "consecutive_failures_current",
+			Name:      "consecutive_setup_failures",
```

Comment:
```suggestion
			Name:      "consecutive_setup_failures_current",
```

The `_current` suffix is a Prometheus convention because this is a gauge so only shows the current value (vs _total for counters).

Reply:
Done.